src/core/biblio.js

```diff
@@ -93,24 +93,61 @@ export function wireReference(rawRef, target = "_blank") {
   `;
 }
 
-export function stringifyReference(ref) {
+export function stringifyReference(ref,style) {
   if (typeof ref === "string") return ref;
-  let output = `<cite>${ref.title}</cite>`;
-  if (ref.href) {
-    output = `<a href="${ref.href}">${output}</a>. `;
+  let output = "";
+  switch (style) {
+  	case "APA": 
```

Comment:
It would be great to move each case into its own function, like:

```JS
case "APA":
  output = entryToAPA(entry);
  break;
case "MLA": 
  output = entryToMLA(entry);
  break;
default:
   output = entryToW3C(entry);
return output; 
```

And then:

```JS
function entryToAPA(entry) {
   // logic here...
}
```

Makes it easier to test and maintain.